acpi/src/madt.rs

```diff
@@ -578,19 +619,20 @@ pub struct MultiprocessorWakeupEntry {
     mailbox_address: u64,
 }
 
-fn parse_mps_inti_flags(flags: u16) -> Result<(Polarity, TriggerMode), AcpiError> {
+#[cfg(feature = "allocator_api")]
```

Comment:
Why does this need to be feature-gated? Not a huge deal but adds a bit of complexity for a random bit of bit-munging code?

Reply:
This is due to `Polarity` and `TriggerMode` existing inside the `::platform::` module, which isn't imported when the `allocator_api` feature isn't being used.

Reply:
Ah okay, makes sense